docs/src/main/sphinx/connector/delta-lake.md

```diff
@@ -772,9 +772,13 @@ The output of the query has the following history columns:
 * - `version`
   - `BIGINT`
   - The version of the table corresponding to the operation
-* - `timestamp`
+* - `creation_time`
   - `TIMESTAMP(3) WITH TIME ZONE`
   - The time when the table version became active
+* - `timestamp`
```

Comment:
commit_timestamp

Reply:
It seems I update the docs in the wrong place. I am supposed to update the inCommitTimestamp

Reply:
I am not going to touch `timestamp` in this pr, it may can handled in separate pr